nemo/collections/tts/models/vits.py

```diff
@@ -97,14 +97,28 @@ def _setup_normalizer(self, cfg):
                     'text_normalizer.whitelist', cfg.text_normalizer.whitelist
                 )
 
-            self.normalizer = instantiate(cfg.text_normalizer, **normalizer_kwargs)
-            self.text_normalizer_call = self.normalizer.normalize
+            try:
+                import nemo_text_processing
+
+                self.normalizer = instantiate(cfg.text_normalizer, **normalizer_kwargs)
+                self.text_normalizer_call = self.normalizer.normalize
+            except Exception as e:
+                logging.error(e)
+                raise ImportError(
+                    "`nemo_text_processing` not installed, see https://github.com/NVIDIA/NeMo-text-processing for more details"
+                )
```

Comment:
> Should we use the same import guard like in other areas?
> 
> ```
> try:
>     import nemo_text_processing
>     NEMO_TEXT_INSTALLED = True
> except ModuleNotFoundError:
>     NEMO_TEXT_INSTALLED = False
> 
> if not NEMO_TEXT_INSTALLED:
>   raise()
> ```

I like this suggestion! And how about putting this to base.py so that every class can just import NEMO_TEXT_INSTALLED from base.py?

Reply:
This code style is good, but are we really wanna unify it in this PR? This is a known issue that is not applied for `nemo_text_processing`. We have other lib import needs to be updated as well.

Reply:
Let's do a separate PR for that.

Reply:
Sure, we can leave it for a separate PR.

Reply:
Added it to JIRA backlog. Please feel free to pick it up.